### PR TITLE
Startup processing

### PR DESCRIPTION
**Pull request by theis999**
### Changes
- Added drawGraph to solution file.
- Added missing return statements in drawGraph::graph_scatterplot_exec.
- Removed Main::PrintStartArguments.
- Removed Main::SetSettingsFile.
- Removed Main::SetDatasource.
- Added Main::handle_exe_arguments to handle PrintStartArguments, SetSettingsFile & SetDatasource.
- Removed PbPlots examples
- Added new file MenuFunctions.c and moved functions from menu.c over.

### Startup processing
This changes how the startup processing in Main::main is. In particular argc & argv, which represent the input used to call exe file, can now be used set settings file path and in theory also the csv.

### MenuFunctions
This is a bit special. C allows the implementation of a .h file to be split between multiple .c files. Which i have used to split the pure menu handling of Menu.c and the specialized actions of the `*functions(void)` array.
This is mostly a semantic change.

### CODE REVIEW

File: PowerHouse/drawGraph.c [hunk 1]
```diff
@@ -136,6 +137,7 @@ int graph_scatterplot_exec(DataType type, Datapoint *data, time_t day)
             fprintf(stderr, "%c", errmsg->string[i]);
         }
         fprintf(stderr, "\n");
+        return 0;
```
Comment (chrnigi) [inline PowerHouse/drawGraph.c]: Should return 1 in error case to be consistent with returns on line 57 and line 112.

File: PowerHouse/drawGraph.c [hunk 2]
```diff
@@ -127,6 +127,7 @@ int graph_scatterplot_exec(DataType type, Datapoint *data, time_t day)
         double *pngdata = ConvertToPNG(&length, canvasref->image);
         WriteToFile(pngdata, length, "graph.png");
         DeleteImage(canvasref->image);
+        return 1;
```
Comment (chrnigi) [inline PowerHouse/drawGraph.c]: Should return 0 here, or at least be the opposite of error returns.

**Review by chrnigi — APPROVED** [review]
Make returns consistent in `graph_scatterplot_exec`, else ok.

File: PowerHouse/drawGraph.c (hunk 2 above)
Comment (theis999) [reply PowerHouse/drawGraph.c]: OK

**Pull request merged**